SharedSource/Core/ReSharperAnnotations/MustUseReturnValueAttribute.cs

```diff
@@ -63,6 +63,7 @@
 // ReSharper disable InconsistentNaming
 // ReSharper disable once CheckNamespace
 
+#nullable disable

```

Comment:
do have have to change JetBrains.Annotation source?

Reply:
There are nullability errors if we don't disable here and i didn't want to change the source too much. I don't think its possible to disable files from the csproj

Reply:
i.c. okay ( I think)